common/alert-message.js

```diff
@@ -18,6 +18,12 @@ module.exports = function alertMessage({ locale, status }) {
                 cy: ``
             });
             break;
+        case 'applicationDeleted':
```

Comment:
I don't thinks this file should be moved into common.  This block doesn't need to be in here, should probably assign this value directly in `dashboard.js` and keep this file as a helper for the user controller. If we have more message that need to be shared for the form-router code add a similar `alert-message.js` helper in there. Keen to keep `common/` for code that is genuinely site-wide.

Reply:
Fair enough, have moved it back.